### PR TITLE
Add stopgap for unaggregated keys

### PR DESCRIPTION
**Pull request by jessicayuen**
Log error and close the watch.

For longer term, there is an open issue to support unaggregated keys.

Signed-off-by: Jess Yuen <jyuen@lyft.com>

### CODE REVIEW

File: internal/app/orchestrator/orchestrator.go [hunk 1]
```diff
@@ -137,22 +132,17 @@ func (o *orchestrator) CreateWatch(req transport.Request) (transport.Watch, func
...
 			"error", err,
 			"request_type", req.GetTypeURL(),
-			"request", req,
-		).Warn(ctx, "failed to map to aggregated key")
-		// Mimic the aggregated key.
-		// TODO (https://github.com/envoyproxy/xds-relay/issues/56). Can we
-		// condense this key but still make it granular enough to uniquely
-		// identify a request?
-		if req.GetRaw().V2 != nil {
-			aggregatedKey = fmt.Sprintf("%s%s", unaggregatedPrefix, req.GetRaw().V2)
```
Comment (jyotimahapatra) [inline internal/app/orchestrator/orchestrator.go:152]: Can you check if the `GetRaw` function is still used in other places and remove it if its not.

File: internal/app/orchestrator/orchestrator_test.go [hunk 2]
```diff
@@ -177,6 +177,35 @@ func TestGoldenPath(t *testing.T) {
...
+		},
+	)
+	// assert this request will not map to an aggregated key.
+	_, err := mapper.GetKey(req)
+	assert.Error(t, err)
+
+	respChannel, _ := orchestrator.CreateWatch(req)
+	testutils.AssertCounterValue(t, mockScope.Snapshot().Counters(),
+		fmt.Sprintf("mock_orchestrator.watch.errors.unaggregated_key"), 1)
+	assert.NotNil(t, respChannel)
```
Comment (jyotimahapatra) [inline internal/app/orchestrator/orchestrator_test.go:205]: is it possible to check the watch is closed?

**Review by jyotimahapatra — COMMENTED** [review]
lgtm, some questions.

File: internal/app/orchestrator/orchestrator.go (hunk 1 above)
Comment (jessicayuen) [reply internal/app/orchestrator/orchestrator.go:152]: Still used here unfortunately: https://github.com/envoyproxy/xds-relay/blob/85bc48c1fffae9a13e7c41a5d572d6eeec48496a/internal/app/orchestrator/orchestrator.go#L163 :(

Comment (jyotimahapatra) [reply internal/app/orchestrator/orchestrator.go:152]: i think this is a mistake from my previous refactoring. There should be handling for both v2/v3 here. I will fix it in a separate PR

**Review by jyotimahapatra — APPROVED** [review]

**Pull request merged**